.github/scripts/js/e2e/slash_workflow_comand.js

```diff
@@ -0,0 +1,188 @@
+const {abortFailedE2eCommand, knownLabels, knownProviders} = require("../constants");
+const {parseCommandArgumentAsRef} = require("../ci");
+
+/**
+ * Build valid return object
+ * @param {string} flag for realease issue logic
+ * @param {object} workflow_id - workflow id
+ * @param {object} targetRef - workflow target ref
+ * @param {object} inputs - target workflow inputs
+ * @return {object}
+ */
+function buildReturn(flag, workflow_id, targetRef, inputs) {
+  return {
+    [flag]: true,
+    workflow_id,
+    targetRef,
+    inputs,
+  }
+}
+
+/**
+ * Try parse e2e abort arguments
+ * @param {object} inputs
+ * @param {object} inputs.core - A reference to the '@actions/core' package.
+ * @param {object} inputs.github - A pre-authenticated octokit/rest.js client with pagination plugins.
+ * @param {object} inputs.context - A reference to context https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts#L6
+ * @param {string} inputs.argv - array of slash command argv[0] is commnad
+ * @return {object}
+ */
+function tryParseAbortE2eCluster({argv, context, core, github}){
+  const command = argv[0];
+  if (command !== abortFailedE2eCommand) {
+    return null;
+  }
+
+  if (argv.length !== 8) {
+    let err = 'clean failed e2e cluster should have 6 arguments'
+    switch (argv.length){
+      case 7:
+        err = 'comment id for starting e2e is required';
+        break;
+      case 6:
+        err = 'cluster_prefix and comment id for starting e2e are required';
+        break;
+      case 5:
+        err = 'artifact name and cluster_prefix and comment id for starting e2e are required';
+        break;
+      case 4:
+        err = 'run id, artifact name and cluster_prefix and comment id for starting e2e are required';
+        break;
+      case 3:
+        err = 'ran for (provider, layout, cri, k8s version), run id, artifact name and cluster_prefix and comment id for starting e2e are required';
+        break;
+      case 2:
+        err = 'pull_request_ref, ran for (provider, layout, cri, k8s version), run id, artifact name, and cluster_prefix and comment id for starting e2e are required';
+        break;
+      case 1:
+        err = 'ci_commit_ref_name, pull_request_ref, ran for (provider, layout, cri, k8s version), run id, artifact name, and cluster_prefix and comment id for starting e2e are required';
+        break;
```

Comment:
Deducing missing parameters from length is inaccurate. I think it is enough to display one message with words like "make sure all required arguments are present".

Reply:
i've remove it